Resources/Prototypes/Entities/Constructible/Furniture/instruments.yml

```diff
@@ -69,17 +58,4 @@
   - type: Instrument
     program: 20
   - type: Sprite
-    sprite: Objects/Fun/Instruments/otherinstruments.rsi
     state: church_organ
-
-- type: entity
-  name: xylophone
-  parent: BasePlaceableInstrument
-  id: XylophoneInstrument
-  description: Rainbow colored glockenspiel.
-  components:
-  - type: Instrument
-    program: 13
-  - type: Sprite
-    sprite: Objects/Fun/Instruments/otherinstruments.rsi
-    state: xylophone
```

Comment:
and the xylophone is yet to be seen again

Reply:
I changed it to be handheld.